scripts/levels/base.gd

```diff
@@ -27,7 +27,7 @@ func _ready():
 		random_time = rand_range(2,5)
 		random_enemy = rand_range(0,2)
 		random_height = rand_range(0, viewport_height / 2)
-		yield(get_tree().create_timer(random_time),"timeout")
+		$Timer.start(random_time); yield($Timer, "timeout")
```

Comment:
@ThomasChr would you mind putting the `yield()` call into a separate line for better readability?

Reply:
I could do that. Of course. That was the proposed stackoverflow solution and I find it rather intriguing.

Reply:
Feel free to change it, otherwise I will hopefully think of it the next days. Or make an issue so it won‘t be forgotten.